.github/workflows/deploy.yml

```diff
@@ -145,32 +145,16 @@ jobs:
 
       # On branches the ticket number is used as image tag. To ensure the pods are restarted
       # and pull the the latest image, we uninstall the helm release first. (Otherwise same tag -> no change)
-      - name: Uninstall dbildungs_iam_client Helm chart (dev branch)
-        if: ${{ inputs.dbildungs_iam_client_branch != 'main' }}
+      # We also need to uninstall all pods to savely recreate the database
+      - name: Uninstall all helm-charts in the namespace
```

Comment:
is this and the extra line in the configmap only temporarily to transfere everything?